apps/chat/src/store/share/share.epics.ts

```diff
@@ -780,4 +866,9 @@ export const ShareEpics = combineEpics(
 
   triggerGettingSharedListingsConversationsEpic,
   triggerGettingSharedListingsPromptsEpic,
+
+  deleteSharedConversationFolderEpic,
+  deleteSharedPromptFolderEpic,
+  renameSharedConversationFolderEpic,
+  renameSharedPromptFolderEpic,
```

Comment:
what's difference between `delete` and `rename`? Do we really need pair epics for these cases?

Reply:
the difference is payload and actions on what they triggered, that's why we need separate actions